iso15118/secc/transport/udp_server.py

```diff
@@ -99,6 +100,7 @@ async def start(self):
             f"{SDP_MULTICAST_GROUP}%{self.iface} "
             f"and port {SDP_SERVER_PORT}"
         )
+        self.ready_event.set()
```

Comment:
I'm not sure if there is much value in making read_event a member variable as the only thing time it is used for is to indicate when the udp server is setup? Wouldn't it suffice to pass it through the start() method?

Reply:
Done.